tensorflow_addons/metrics/README.md

```diff
@@ -39,3 +42,6 @@ must:
 
 #### Documentation Requirements
  * Update the table of contents in this sub-package's README.
+
+
+[zhang and zhou 2014]: https://ieeexplore.ieee.org/document/6471714
```

Comment:
This breaks with the way we've done READMEs. Mind just adding the links in the table for standardization? Your way reduces some character count but I'd rather not confuse future contributors.

Reply:
done